proto/commons/massa/model/v1/address.proto

```diff
@@ -28,7 +28,13 @@ message NativeAddress {
   // Address category
   AddressCategory category = 1;
   // Address version
-  fixed64 version = 2;
+  uint64 version = 2;
```

Comment:
on abi side, after a quite long discussion you may remember, we decided that AddressCategory should be an enum

would that lead to a problem?
```protobuf
enum AddressCategory {
   // Unspecified address category
   ADDRESS_CATEGORY_UNSPECIFIED = 0;
   // User address
   ADDRESS_CATEGORY_USER_ADDRESS = 1;
   // Smart contract address
   ADDRESS_CATEGORY_SC_ADDRESS = 2;
 }
```

Reply:
Nope, we don't use NativeAddress in API side as It's a string

Reply:
fine :)